alpha/declcfg/declcfg_to_model.go

```diff
@@ -168,6 +168,70 @@ func ConvertToModel(cfg DeclarativeConfig) (model.Model, error) {
 		}
 	}
 
+	// deprecationsDuplicateCheck tracks the set of package names
```

Comment:
We missed this comment yesterday when we changed the variable name.
```suggestion
	// deprecationsByPackage tracks the set of package names
```